apps/monarch/monarch-frontend/src/app/SearchTherapists.tsx

```diff
@@ -177,7 +195,22 @@ const EditButton: React.FC<{
 
   const handleSave = () => {
     if (isFormValid()) {
-      // TODO: save data in database
+      const updatedInfo: Practitioner = {
+        uuid: therapist.uuid,
+        phoneNumber: phone,
+        website: website || '',
+        modality: therapyType,
+        businessLocation: address,
+        businessName: businessName,
+        minAgeServed: therapist.minimumAgeServed,
+        email: email,
+        fullName: fullName,
+        languagesList: languages.map((lang) => lang.value),
+        geocode: therapist.geocode,
+        dateJoined: dateJoined,
+        familiesHelped: familiesHelped,
+      };
+      controller.updateTherapist(updatedInfo, accessToken);
```

Comment:
Right now, when someone finishes editing the form, the changes aren't reflected until the user manually refreshes. Can we add a call to `setReload(true)` so we automatically refresh after submitting?